src/compiler/raw-file.js

```diff
@@ -11,20 +11,27 @@ export default class RawFileCompiler {
 
     static _createTestFn (commands) {
         return async testRun => {
-            for (var i = 0; i < commands.length; i++) {
-                var callsite = commands[i] && commands[i].callsite;
-                var command  = null;
+            var callsite        = null;
+            var commandPromises = commands.map(commandObject => {
+                callsite = commandObject && commandObject.callsite;
```

Comment:
Why do you check that `commandObject` exists? Is it possible that some of commands from the array is null or undefined?